app/assets/javascripts/services_renderers_enablers.js.es6

```diff
@@ -34,6 +34,16 @@ function voteIdeaQuality(inputName){
   });
 }
 
+function updateIdeaContent(element, $object, updateContent, ideaId, errorMessage) {
+  if (element === ".idea-title") {
+    $object.parent().data().title = updateContent;
+  } else {
+    $object.parent().data().body = updateContent;
+  }
```

Comment:
Question: Seems like this if statement is built to handle the case of updating either the title or the body of an idea. It might introduce some duplication, but is it easier to have these actually be separate methods? Or to pull this 'if' out to its own method? I understand what's happening here now, but pretty sure it'll take me a little bit to figure it out again six months from now.